packages/api/src/builder/routes.ts

```diff
@@ -79,6 +106,17 @@ export function getReqSerializers(config: ChainForkConfig): ReqSerializers<Api,
       },
     },
     submitBlindedBlock: getBeaconReqSerializers(config)["publishBlindedBlock"],
+    getExpectedWithdrawals: {
+      writeReq: (state_id, proposal_slot) => ({
+        params: {state_id},
+        query: {proposal_slot},
+      }),
+      parseReq: ({params, query}) => [params.state_id, query.proposal_slot ?? 0],
```

Comment:
Do you need a default value for proposal slot if you have defined it as `Slot | undefined`?